File: src/exemplo01/Endereco.java
package exemplo01;

public class Endereco {
    private final String rua;
    private final int numero;

    public Endereco(String rua, int numero) {
        this.rua = rua;
        this.numero = numero;
    }

    public String getRua() {
        return rua;
    }

    public int getNumero() {
        return numero;
    }
}

File: src/exemplo01/Endereco.kt
package exemplo01.kt

class Endereco {
    private var rua: String
    private var numero: Int

    constructor(rua :String, numero :Int) {
        this.rua = rua
        this.numero = numero
    }

    fun getRua(): String {
        return rua
    }

    fun getNumero(): Int {
        return numero
    }
}
